src/node/mod.rs

```diff
@@ -1879,4 +1898,98 @@ mod tests {
             .await
             .unwrap()
     }
+
+    async fn test_sync_plot_inner() {
+        crate::utils::test_init();
+
+        let node_span = tracing::trace_span!("node 1");
+        let dir = TempDir::new().unwrap();
+        let chain = chain_spec::dev_config().unwrap();
+        let node = Node::dev()
+            .role(Role::Authority)
+            .dsn(DsnBuilder::dev().listen_addresses(vec!["/ip4/127.0.0.1/tcp/0".parse().unwrap()]))
+            .network(
+                NetworkBuilder::dev()
+                    .listen_addresses(vec!["/ip4/127.0.0.1/tcp/0".parse().unwrap()]),
+            )
+            .role(Role::Authority)
+            .build(dir.path(), chain.clone())
+            .instrument(node_span.clone())
+            .await
+            .unwrap();
+        let (plot_dir, cache_dir) = (TempDir::new().unwrap(), TempDir::new().unwrap());
+        let farmer = Farmer::builder()
+            .build(
+                Default::default(),
+                node.clone(),
+                &[PlotDescription::new(plot_dir.as_ref(), bytesize::ByteSize::mib(40)).unwrap()],
```

Comment:
if we change the minimum size, this test may fail automatically. Maybe you can turn this magic number into a constant, so it will be more obvious to you remember changing the constant instead of some numbers in the code?